src/components/policy/policy_external/test/sql_pt_ext_representation_test.cc

```diff
@@ -57,6 +59,37 @@ namespace test {
 namespace components {
 namespace policy_test {
 
+namespace {
+const std::string kSdlPreloadedPtJson = "json/sdl_preloaded_pt.json";
+}  // namespace
+
+policy_table::Table LoadPreloadedPT(const std::string& filename) {
+  std::ifstream ifile(filename);
+  EXPECT_TRUE(ifile.good());
+  Json::CharReaderBuilder reader_builder;
+  Json::Value root(Json::objectValue);
+  Json::parseFromStream(reader_builder, ifile, &root, nullptr);
+  root["policy_table"]["module_config"].removeMember("preloaded_pt");
```

Comment:
@ychernysheva do you really need this removal here? It looks like workaround
Maybe it is better to remove this field from a test file instead?

Reply:
We need to remove this field to get valid preloaded pt for update (requirements say that such table cannot have preloaded_pt member, and validation check will fail if this row is present).
We cannot delete this row from initial file because it is a copy of current preloaded pt from AppMain folder, which is copied when SDL builds. And this file is used not only for PTU simulation here, for initialization of preloaded Pts too (for example, in UTs for CacheManager and PolicyManager testing). So this file is shared.
So I propose to left it as is.